Services/RelationService.cs

```diff
@@ -0,0 +1,179 @@
+﻿using AndersenCoreApp.Interfaces.Repositories;
+using AndersenCoreApp.Interfaces.Services;
+using AndersenCoreApp.Models.DomainModels;
+using AndersenCoreApp.Models.ModelsDTO;
+using AndersenCoreApp.Models.ViewModels;
+using AutoMapper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AndersenCoreApp.Services
+{
+    public class RelationService : IRelationService
+    {
+        private IRelationRepository relationRepo;
+        private ICountryRepository countryRepo;
+        private IMapper ViewModelMapper;
+
+        public RelationService(IRelationRepository relations, ICountryRepository countries)
+        {
+            relationRepo = relations;
+            countryRepo = countries;
+            ViewModelMapper = ConfigureMapperForViewModel();
+        }
+
+        public bool ContainsRelation(Guid id)
+        {
+            return relationRepo.Any(id);
+        }
+
+        public RelationViewModel GetOne(Guid id)
+        {
+            var relation = relationRepo.GetOne(id);
+            return ViewModelMapper.Map<Relation, RelationViewModel>(relation);
+        }
+
+        public IQueryable<Relation> GetAll()
+        {
+            return relationRepo.GetAll();
+        }
+
+        //TODO write an algorithm for checking postal mask
+        public bool CheckPostalMask(Relation relation)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void Create(RelationViewModel relation)
+        {
+            if (relation != null)
+            {
+                var relationToCreate = CreateRelationFromRelationViewModel(relation);
+                relationRepo.Create(relationToCreate);
+            }
+        }
+
+        public void Update(RelationViewModel relation)
+        {
+            if (relation != null)
+            {
+                var relationToUpdate = CreateRelationFromRelationViewModel(relation);
+                relationRepo.Update(relationToUpdate);
+            }
+        }
+
+        public void Delete(params Guid[] identificators)
+        {
+            relationRepo.Delete(identificators);
+        }
+
+        public List<RelationViewModel> GetSortedAndFilteredRelations(IQueryable<Relation> list, string categoryName, string orderProperties)
```

Comment:
IQueryable<Relation> is not a list. Is query, so name it like query, relations or queryRelations.